src/main/kotlin/minesweeper/Runner.kt

```diff
@@ -0,0 +1,14 @@
+package minesweeper
+
+import minesweeper.domain.BoomPositionMaker
+import minesweeper.domain.MineMap
+import minesweeper.view.InputView
+import minesweeper.view.OutputView
+
+fun main(args: Array<String>) {
```

Comment:
코틀린 1.3부터 `main()`를 매개 변수 없이 사용할 수 있습니다.

```suggestion
fun main() {
```